test/client/api.js

```diff
@@ -427,4 +426,24 @@ ClientApi.heartbeat = function (origin) {
   return (new ClientApi(origin)).doRequest('GET', origin + '/__heartbeat__')
 }
 
+function getQueryString (options) {
+  var qs = ''
```

Comment:
The reason I didn't start with a default `'?'` is so that nothing gets added in the no-options condition. Not that a trailing `?` would do any harm of course, but why add one to the URL if there is no need to?

Using `querystring.stringify()` would behave differently in a few ways:

* We'd need to `!! keys` to force it to a boolean.
* We'd be setting `keys=false` which, presumably, would do no harm (although I don't know enough to be certain).
* We'd add empty params for any options that are `null`, `undefined` or the empty string (again no harm, but pretty unnecessary).

Altogether, it felt simpler to maintain the pre-existing approach of doing it manually.

Having said all that, it's not something I'm fussed about so I'm happy to make the change if you prefer. Just wanted to put the above out there in case it affects your decision.